elements/lisk-chain/src/slots.ts

```diff
@@ -13,45 +13,47 @@
  */
 
 interface SlotsInput {
-	readonly epochTime: string;
+	readonly genesisBlockTimestamp: number;
 	readonly interval: number;
 }
 
 const SEC_IN_MS = 1000;
 
 export class Slots {
-	private readonly _epochTime: Date;
+	private readonly _genesisTime: Date;
```

Comment:
Not important, but I think it was originally `Date` because the epochTime input was ISO date.
It seems it will be a lot simpler if we use number (timestamp) here

Reply:
Changed.